doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-Linux/index.md

```diff
@@ -2,304 +2,185 @@
+            kaa_client_stop(context);
+        }
 
-    2. For manual installation, refer to the following example:
+        int main(void)
+        {
+            kaa_client_t *kaa_client = NULL;
+            kaa_error_t error = kaa_client_create(&kaa_client, NULL);
+            if (error) {
+                return EXIT_FAILURE;
+            }
 
-       ~~~
-       $ sudo apt-get install libbz2-dev libbz2-1.0 zlib1g zlib1g-dev
-       $ wget http://sourceforge.net/projects/boost/files/boost/1.58.0/boost_1_58_0.tar.gz
-       $ tar -zxf boost_1_58_0.tar.gz
-       $ cd boost_1_58_0/
-       $ ./bootstrap.sh
-       $ sudo ./b2 install
-       ~~~
+            error = kaa_client_start(kaa_client, dummy_function, (void *)kaa_client, 0);
 
-4. Install the [**AvroC++**](http://avro.apache.org/docs/1.7.6/api/cpp/html/index.html) library manually:
+            kaa_client_destroy(kaa_client);
 
-   ~~~
-   $ wget http://archive.apache.org/dist/avro/avro-1.7.5/cpp/avro-cpp-1.7.5.tar.gz
-   $ tar -zxf avro-cpp-1.7.7.tar.gz
-   $ cd avro-cpp-1.7.5/
-   $ cmake -G "Unix Makefiles"
-   $ sudo make install
-   ~~~
+            if (error) {
+                return EXIT_FAILURE;
+            }
 
-5. Install the [**Botan**](http://botan.randombit.net/) library by executing the following command:
+            return EXIT_SUCCESS;
+        }
 
-   ~~~
-   $ wget https://github.com/randombit/botan/archive/1.11.28.tar.gz
-   $ tar -zxf 1.11.28.tar.gz
-   $ cd botan-1.11.28/
-   $ ./configure.py
-   $ sudo make install
-   $ sudo ln -s /usr/local/include/botan-1.11/botan /usr/local/include/botan
-   ~~~
+    The application code is simple: application just prints the string *"Hello, I am a Kaa Application!\n"*. That's it.
 
-6. Install the **[SQLite](https://www.sqlite.org/index.html)** library by executing the following command:
-    1. For automatic installation, execute the following commands (tested on Ubuntu 14.04):
+4. Create `CMakeLists.txt` file in the `my-kaa-application` directory. It is a top-level cmake file which is responsible for the application build.
 
-       ~~~
-       $ sudo apt-get install libsqlite3-0 libsqlite3-dev
-       ~~~
+        touch CMakeLists.txt
 
-    2. For manual installation, refer to the following example:
+    Add the following code to top-level `CMakeLists.txt`
 
-       ~~~
-       $ wget https://www.sqlite.org/2015/sqlite-autoconf-3081002.tar.gz
-       $ tar -zxf sqlite-autoconf-3081002.tar.gz
-       $ cd sqlite-autoconf-3081002/
-       $ ./configure
-       $ sudo make install
-       ~~~
+        cmake_minimum_required(VERSION 3.5.2)
+        project(kaa-application C)
 
-**NOTE:** Instead of manually installing all required components and libraries, you can follow [the quick way to build C/C++ endpoint SDK](#Linux-Quickway).  (only applicable for x86\_64 platform build)
+    Set minimum cmake version which is required to build kaa-application project.The second line sets project name and the language of the project.
```

Comment:
The first line sets the minimum cmake version required to build kaa-application project.